packages/docs/PropTables/StatefulTablePropTable.tsx

```diff
@@ -36,6 +36,12 @@ const statefulTableProps: Prop[] = [
     defaultValue: 'false',
     description: 'Defines if table should be paginated.',
   },
+  {
+    name: 'search',
+    types: 'boolean',
+    defaultValue: 'false',
+    description: 'Defines if table should have search',
```

Comment:
Can we expand upon this? Like what fields we are searching over?

Reply:
Is it okay? 
`Defines if table should have search by visible fields with type string, number, boolean`